pkg/apim/APIMUtility.go

```diff
@@ -51,3 +66,21 @@ func (apim *Bundle) IsPayloadJSON() bool {
 	var js json.RawMessage
 	return json.Unmarshal([]byte(apim.Payload), &js) == nil
 }
+
+func (odataFilters *OdataParameters) InitOdataQuery() (string, error) {
+
+	v := reflect.ValueOf(odataFilters).Elem()
+	typeOfS := v.Type()
```

Comment:
What does S stand for? Could you think of a better var name? Or maybe even `t` is suffice, although personally I don't like one letter var name, which are however very common in go.

Reply:
fixed